### DISCUSSION TITLE
ESP32 is reset when WIFI is interrupted

### DISCUSSION THREAD

**Discussion by vowovrz** [schreibfaul1/ESP32-audioI2S#260] [General]
#include "Arduino.h"
#include "WiFi.h"
#include "Audio.h"
#define I2S_DOUT     33
#define I2S_BCLK      25
#define I2S_LRC        26
Audio audio;
String ssid =    "********";
String password = "********";
void setup() {
  WiFi.disconnect();
  WiFi.mode(WIFI_STA);
  WiFi.begin(ssid.c_str(), password.c_str());
  while (WiFi.status() != WL_CONNECTED)
  delay(1500);
  audio.setPinout(I2S_BCLK, I2S_LRC, I2S_DOUT);
  audio.setVolume(100);
  audio.connecttohost("********");
}
void loop()
{
  audio.loop();
}

------------------------------------

06:12:54.018 -> Guru Meditation Error: Core  1 panic'ed (LoadProhibited). Exception was unhandled.
06:12:54.018 -> Core 1 register dump:
06:12:54.052 -> PC      : 0x400d1670  PS      : 0x00060830  A0      : 0x800d1738  A1      : 0x3ffb1f00  
06:12:54.052 -> A2      : 0x00000000  A3      : 0x3ffd7331  A4      : 0x000000e6  A5      : 0x00000001  
06:12:54.052 -> A6      : 0x00000000  A7      : 0x00000000  A8      : 0x00000000  A9      : 0x00000000  
06:12:54.052 -> A10     : 0x23140000  A11     : 0x21000000  A12     : 0x3ffc3b38  A13     : 0x3ffc1a38  
06:12:54.052 -> A14     : 0x045efe55  A15     : 0x00000040  SAR     : 0x0000000a  EXCCAUSE: 0x0000001c  
06:12:54.086 -> EXCVADDR: 0x00000008  LBEG    : 0x4000c2e0  LEND    : 0x4000c2f6  LCOUNT  : 0xffffffff  
06:12:54.086 -> 
06:12:54.086 -> ELF file SHA256: 0000000000000000
06:12:54.086 -> 
06:12:54.086 -> Backtrace: 0x400d1670:0x3ffb1f00 0x400d1735:0x3ffb1f20 0x400dae56:0x3ffb1f40 0x400db29f:0x3ffb1f70 0x400d13ce:0x3ffb1f90 0x400ec5b8:0x3ffb1fb0 0x4008963e:0x3ffb1fd0
06:12:54.086 -> 

----------------------------------------------

Decoding stack results
0x400d1670: WiFiClientRxBuffer::read(unsigned char*, unsigned int) at C:\Users\Administrator\AppData\Local\Arduino15\packages\esp32\hardware\esp32\1.0.6\libraries\WiFi\src\WiFiClient.cpp line 107
0x400d1735: WiFiClient::read(unsigned char*, unsigned int) at C:\Users\Administrator\AppData\Local\Arduino15\packages\esp32\hardware\esp32\1.0.6\libraries\WiFi\src\WiFiClient.cpp line 437
0x400dae56: Audio::processWebStream() at C:\Users\Administrator\Documents\Arduino\libraries\ESP32-audioI2S-master\src\Audio.cpp line 3132
0x400db29f: Audio::loop() at C:\Users\Administrator\Documents\Arduino\libraries\ESP32-audioI2S-master\src\Audio.cpp line 2238
0x400d13ce: loop() at C:\Users\Administrator\Desktop\sketch_jan19a/sketch_jan19a.ino line 22
0x400ec5b8: loopTask(void*) at C:\Users\Administrator\AppData\Local\Arduino15\packages\esp32\hardware\esp32\1.0.6\cores\esp32\main.cpp line 23
0x4008963e: vPortTaskWrapper at /home/runner/work/esp32-arduino-lib-builder/esp32-arduino-lib-builder/esp-idf/components/freertos/port.c line 143

**Comment by schreibfaul1** [comment]
Difficult to say, maybe the selected board doesn't match the one actually used or the partition.csv doesn't match the board
